scripts/test/console_output_test.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+
+import os
+import sys
+import pexpect
+
+def main():
+	if len(sys.argv) == 1:
+		print ("Usage: ./scripts/test/console_output_test.py <console_path>")
+		sys.exit(1)
+
+	if not os.path.isdir("resources/test_data/tbl"):
+		print ("Cannot find resources/test_data/tbl. Are you running the test suite from the main folder of the Hyrise repository?")
+		sys.exit(1)
+
+	console_path = sys.argv[1]	
+	console = pexpect.spawn(console_path + "/hyriseConsole", timeout=15, dimensions=(1920,1080))
```

Comment:
"dimensions" refers to columns/rows - 1920x1080 is a bit much here, my terminal has a default size of 200x64, e.g.. I'd change this to something more realistic.